### ISSUE TITLE
Make the change of PATH persistent

### ISSUE THREAD

**Issue by girone** [danielkaiser/python-chromedriver-binary#2]
This would be quite handy, if it could be used to set up the chromedriver system wide. Unfortunately, it only changes the PATH in the current process. Is there a way to make the change available outside of Python?

See also this [the second and third-placed answers to this SO question](https://stackoverflow.com/a/3952557/841567).

**Comment by danielkaiser** [comment]
Thank you for your suggestion.

As far as I know executing `python` (at least on unix-based os) spawns a new process that has a copy of the environment variables of the parent process but it cannot modify the variables of the parent process. So as far as I can see there is no possibility to change the PATH in the calling proccess.

I think there are two workarounds that might resolve this issue:

- Adding a Python function that appends an `export PATH=$PATH:...` to the users ~/.profile file so it is always set when starting a new shell. 
- Installing a small bash script during `pip install` that exports the PATH. This script then needs to be run with `source name-of-script` or `. name-of-script` to be able to set environment variables of the calling process

In my opinion the first option is not very appropiate because it easily messes up your .profile (e.g. it will not clean up the .profile file on `pip uninstall`). As far as I'm concerned the second option is the best way of setting the PATH variable outside the Python process.

What do you think?

**Comment by girone** [comment]
This might be a completely different approach, but what about not changing the PATH at all but instead symlinking to the binary at `/usr/local/bin/` during setup (for instance, I am not sure which would be the preferred location that is in the PATH by default)?

**Comment by danielkaiser** [comment]
On my machine (debian 9) for example the default PATH is set to `/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games`. None of these directories is user-writeable by default. Therefore copying or symlinking to either of these directories would require higher permissions (root user or in some cases `staff` group membership) to install the package.

I would definitively prefer a solution that does not need special user permissions to install and ideally does not permanently modify the executable binaries on the system.

**Comment by danielkaiser** [comment]
Please have a look at 83981298e841a7a18702a86338b9d54baf97cae3.
I have added a function to print the complete PATH (previous PATH + chromedriver binary location) on the console and a script `chromedriver-path` that calls it. So you can just run
```bash
export PATH=`chromedriver-path`
```
in the shell and the binary should be usable.

What do you think about this solution?

**Comment by girone** [comment]
That's a great idea! 

IMO it is easier to use when `chromedriver-path` returns only the path of chromedriver, and lets the caller decide how to use it. E.g., one would use it like so:

```
export PATH=$PATH:`chromedriver-path`
```

This way there is less "hidden magic" going on and it also supports workflows/OSs which do not use PATH but other environment variables.

**Comment by danielkaiser** [comment]
That is a good argument. I modified the code so that it only prints the path to the binary file. I released it as version v2.35.1 (24a3413) on github and PyPI.
Thank you for your help.
